composer.json

```diff
@@ -63,7 +63,7 @@
         "drupal/token": "^1.5",
         "drupal/view_unpublished": "^1.0",
         "drupal/views_bulk_edit": "^2.5",
-        "drupal/views_bulk_operations": "^3.10",
+        "drupal/views_bulk_operations": "^4.2",
```

Comment:
Version 3 no longer supported.  
Tested version 4 and no functionality seems to have been lost.